### PR TITLE
mail, mailx, mesg, mmv, monop, most, mscore: add link

### PR DESCRIPTION
**Pull request by Sadeeed**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

#6062

### CODE REVIEW

File: pages/common/mail.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > The command operates on the user's mailbox if no argument is given.
 > To send an email the message body is built from standard input.
+> More information: <https://linux.die.net/man/1/mail>.
```
Comment (navarroaxel) [inline pages/common/mail.md:5]: We prefer `manned.org` over `linux.die.net`
```suggestion
> More information: <https://manned.org/mail>.
```

File: pages/common/mailx.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # mailx
 
 > Send and receive mail.
+> More information: <https://linux.die.net/man/1/mail>.
```
Comment (navarroaxel) [inline pages/common/mailx.md:4]:
```suggestion
> More information: <https://manned.org/mailx>.
```

File: pages/common/mesg.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 > Check or set a terminal's ability to receive messages from other users, usually from the write command.
 > See also `write`.
+> More information: <https://manpages.ubuntu.com/manpages/impish/en/man1/mesg.1.html>.
```
Comment (navarroaxel) [inline pages/common/mesg.md:5]:
```suggestion
> More information: <https://manned.org/mesg>.
```

File: pages/common/mmv.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 # mmv
 
 > Move and rename files in bulk.
+> More information: <https://manpages.ubuntu.com/manpages/impish/en/man1/mmv.1.html>.
```
Comment (navarroaxel) [inline pages/common/mmv.md:4]:
```suggestion
> More information: <https://manned.org/mmv.1>.
```

File: pages/common/monop.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # monop
 
 > Finds and displays signatures of Types and methods inside .NET assemblies.
+> More information: <https://manpages.ubuntu.com/manpages/impish/en/man1/monop.1.html>.
```
Comment (navarroaxel) [inline pages/common/monop.md:4]:
```suggestion
> More information: <https://manned.org/monop>.
```

File: pages/common/most.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 # most
 
 > Open one or several files for interactive reading, allowing scrolling and search.
+> More information: <https://manpages.ubuntu.com/manpages/impish/en/man1/most.1.html>.
```
Comment (navarroaxel) [inline pages/common/most.md:4]:
```suggestion
> More information: <https://manned.org/most>.
```

File: pages/common/mscore.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 # mscore
 
 > This command is an alias of `musescore`.
+> More information: <https://musescore.org/en/handbook/3/command-line-options>.
```
Comment (navarroaxel) [inline pages/common/mscore.md:4]: We can remove language and version, maybe it's always redirected to the latest version.
```suggestion
> More information: <https://musescore.org/handbook/command-line-options>.
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by schneiderl — APPROVED** [review]

**Pull request merged**